tests/backup/backup_license_test.go

```diff
@@ -101,6 +103,7 @@ var _ = Describe("{NodeCountForLicensing}", func() {
 		})
 	})
 	JustAfterEach(func() {
+		defer EndPxBackupTorpedoTest(contexts)
 		ctx, err := backup.GetAdminCtxFromSecret()
 		log.FailOnError(err, "Fetching px-central-admin ctx")
 		SetDestinationKubeConfig()
```

Comment:
This might be out of scope for this PR. SetDestinationKubeConfig throws an error and it needs to be handled.